apps/framework-cli/build.rs

```diff
@@ -0,0 +1,16 @@
+use std::io::Result;
+
+fn main() -> Result<()> {
+    println!("cargo:rerun-if-changed=../../packages/protobuf");
```

Comment:
it's time like this that makes me appreciate gradle's tracking of input/output change

Reply:
😆